app/models/form/lettings/pages/net_income_value_check.rb

```diff
@@ -36,6 +36,6 @@ def questions
   end
 
   def interruption_screen_question_ids
-    %w[incfreq earnings ecstat1]
+    %w[incfreq earnings hhmemb ecstat1 age2 ecstat2 age3 ecstat3 age4 ecstat4 age5 ecstat5 age6 ecstat6 age7 ecstat7 age8 ecstat8]
```

Comment:
Do we hide not routed to questions from the list somewhere?

Reply:
I think when we display the interruption screen we filter for questions that are routed to - so we've got similar things where all the person fields are listed for the soft validation checks around pregnancy.